libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -506,3 +534,97 @@ def test_that_case_sensitivity_does_not_affect_distance_strategy(
     conn.execute(text("use master"))
     conn.execute(text(_DROP_COLLATION_DB_QUERY))
     conn.close()
+
+
+@pytest.mark.parametrize("test_filter, expected_ids", TYPE_1_FILTERING_TEST_CASES)
+def test_sqlserver_with_with_metadata_filters_1(
```

Comment:
nit: avoid 1, 2, 3 in the name of the test - instead give a meaningful name as to understand what do these cases cover for e.g. TYPE_1 filtering is equality checks